helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixAdmin.java

```diff
@@ -1104,6 +1105,40 @@ public void setConfig(HelixConfigScope scope, Map<String, String> properties) {
     return _configAccessor.get(scope, keys);
   }
 
+  @Override
+  public void addCustomizedStateAggregationConfig(String clusterName,
```

Comment:
If a user would like to add one type of state or remove one type of state, how to do it? It seems the API is to add or remove the config as a whole, but in most cases, we will use it as a way to add/remove one item in the list view.

Reply:
Fixed. I just Added two new APIs.